### DISCUSSION TITLE
How to prohibit horizontal screen in APP or how to display all content in APP when  the screen is horizontal?

### DISCUSSION THREAD

**Discussion by ccinsz** [beeware/briefcase#1423] [Q&A]
I try to set android:screenOrientation="portrait" in the android\gradle\app\src\main\AndroidManifest.xml, and run "briefcase run android -u -r", but it doesn't work. When the screen is horizontal, some content is not fully displayed, so I want to know how to prohibit horizontal screen in APP? Or how to display all content in APP when  the screen is horizontal?

**Comment by ccinsz** [comment]
How to prohibit horizontal screen in APP？ 

It is solved.

**Reply by mhsmith** [reply]
It would be helpful to other users if you explain how you solved it.

**Reply by ccinsz** [reply]
Put android:screenOrientation="portrait" in the right position,  the activity tag of AndroidManifest.xml and run "briefcase run android -u -r", it can prohibit horizontal screen in APP.
